File: src/main/java/by/ladyka/club/dto/ClubEventTicketPriceDTO.java
package by.ladyka.club.dto;

import by.ladyka.club.entity.EventTicketPriceType;
import lombok.Getter;
import lombok.Setter;

import java.math.BigDecimal;
import java.time.LocalDateTime;

@Setter
@Getter
public class ClubEventTicketPriceDTO {
	private Long id;
	private BigDecimal cost;
	private Long quantity;
	private EventTicketPriceType typePrice;
	private Long eventId;
	private LocalDateTime startActiveTime;
	private LocalDateTime endActiveTime;
	private String modifiedBy;
}

File: src/main/angular/src/app/dto/tickets/clubEventTicketPriceDto.ts
export class ClubEventTicketPriceDto {
  id: number;
  cost: number;
  quantity: number;
  typePrice: string;
  eventId: number;
  startActiveTime: Date;
  endActiveTime: Date;
  modifiedBy: string;
}

export class ClubEventTicketPriceBaseListResultDto {
  items: ClubEventTicketPriceDto[];
  total: number
}
